src/components/POLine/Item/ItemView.js

```diff
@@ -19,76 +19,108 @@ class ItemView extends Component {
     identifierTypes: PropTypes.arrayOf(PropTypes.object),
   }
 
+  static defaultProps = {
+    identifierTypes: [],
```

Comment:
could you please be consistent? Above you've made materialTypes as not required and added default value as `materialTypes = []`. So I've expected to see here the same, but you've introduced a whole attribute defaultProps. 
I understand that we have both in the project, but maybe at least in scope on one PR it could be the same

Reply:
the result is the same for both, but I'd use `defaultProps`

Reply:
Fixed